src/DefaultHandler/LogsFolder.php

```diff
@@ -35,7 +35,11 @@ public static function determineFolder(?string $customFolder = null): ?string
          * them, and package could be fully functional even if failures happen.
          * Silence looks like best option here.
          *
+         * Also for some reason __return_true seems not to be a valid argument¿?
```

Comment:
`¿?` :)

Reply:
it seems that __return_true is not a valid callable https://github.com/vimeo/psalm/issues/3571

I was getting this: 

```
PossiblyInvalidArgument: Argument 1 of set_error_handler expects callable(int, string, string=, int=, array<array-key, mixed>=):bool|null, but possibly different type impure-callable():true provided (see https://psalm.dev/092)
```

Reply:
Sorry, I meant that is strange to find the Spanish `¿?` sign in code instead of the English `?`.

Reply:
changed